docs/index.html

```diff
@@ -80,120 +80,122 @@ <h2>What is the purpose?</h2>
         label: "DDD",
         website: "",
         active: true,
-        moved: 0
+        moved: 0,
+        description:`
+        <p>Domain Driven Design (DDD) is an approach to software development that connects the implementation with an evolving model.</p>
+        <p>We use DDD where appropriate within our codebase to reduce complexity and keep the coding model close to the reality.</p>`
       },
       {
         quadrant: 0,
         ring: 1,
         label: "Functional Programming",
         website: "",
         active: true,
-        moved: 0
+        moved: 0,
+        description:`
+        <p>Functional programming (FP) is a style of programming that treats computation as the evaluation of mathematical functions.
+          Strict FP avoids side-effects and mutable data.
+          </p>
+          <p>We are often inspired by FP and parts of both our front-end and back-end are written in a functional style. We use the lodash/fp 
+            (front-end) and Fugue (back-end) libraries to help us achieve this functional style.</p>
+        `
       },
       {
         quadrant: 1,
-        ring: 0,
-        label: "Looker",
-        website: "",
-        active: true,
-        moved: 0
-      },
-      {
-        quadrant: 1,
-        ring: 0,
-        label: "Airtable",
-        website: "",
-        active: true,
-        moved: 0
-      },
-      {
-        quadrant: 1,
-        ring: 0,
-        label: "Appcues",
-        website: "",
-        active: true,
-        moved: 0
-      },
-      {
-        quadrant: 1,
-        ring: 0,
+        ring: 1,
         label: "Locust",
         website: "",
         active: true,
-        moved: 0
-      },
-      {
-        quadrant: 1,
-        ring: 0,
-        label: "SendGrid",
-        website: "https://sendgrid.com",
-        active: true,
-        moved: 0
+        moved: 0,
+        description: `
+        <p>Locust is an open source load testing framework written in Python.</p>
+        <p>We use Locust to ensure Small Improvements can handle higher than expected traffic and to test potential bottlenecks in our application.</p>
+        `
       },
       {
         quadrant: 1,
         ring: 0,
         label: "Docker",
         website: "https://www.docker.com",
         active: true,
-        moved: 0
+        moved: 0,
+        description:`
+        <p>Docker is a container virtualization service that packages an application and its dependencies in a container that can run on any Linux</p>
```

Comment:
"on any Linux"…? Is there something missing at the end? Maybe it should end with "Linux environment" or "Linux-based system"?